### ISSUE TITLE
Removing handcuffs in admin mode

### ISSUE THREAD

**Issue by talarmin29** [KB-RolePlay/kbrp-issues#157]
### Description

The handcuff removal option is no longer displayed in admin mode.

### Expected behaviour

Display option.

### Reproduction steps

```bash
1. In admin mode, find a person wearing handcuffs
2. Press the C key
```

### Screenshots

![Desktop Screenshot 2023 05 30 - 21 17 26 04](https://github.com/KB-RolePlay/kbrp-issues/assets/21173205/174e4d23-eacf-43d0-a2c0-50821fc26ed5)

**Comment by hugoheml** [comment]
Normaly solved in release `v1.1.14`, we need to test the issue before close them.

**Comment by Kobralost** [comment]
Fixed problem
